client/my-sites/sidebar/index.jsx

```diff
@@ -801,7 +800,7 @@ export class MySitesSidebar extends Component {
 			return null;
 		}
 
-		if ( ! site || ! site.options ) {
+		if ( ! site?.options?.admin_url ) {
```

Comment:
Note that the condition will be met if the URL is an empty string, which is technically a valid URL. That said, it's probably doing the right thing by returning `null` in that situation.

Reply:
Yeah, I figured we'd still have trouble with an empty string, so the intent here was to just give up if there's no actual URL. Thank you for pointing it out, though.